db/seeds/conversations.rb

```diff
@@ -4,15 +4,18 @@
 conversation = Conversation.find_or_create_by!(developer:, business:)
 SeedsHelper.create_message!(conversation:, sender: business, body: "Your XP looks perfect.", cold_message: true)
 SeedsHelper.create_message!(conversation:, sender: developer, body: "Let's chat over Zoom.")
+UpdateDeveloperResponseRateJob.perform_now(developer)
```

Comment:
I'm confused as to why this needs to be performed outside of the normal creating of messages. Should the job get kicked off "in 0 seconds" when running the seeds?

Reply:
I thought it would, but it didn't appear to be doing so.
I shall conduct further experiments.

Reply:
Working as expected now without the call to the job.